File: 2265-partition-array-according-to-given-pivot/2265-partition-array-according-to-given-pivot.py
class Solution:
    def pivotArray(self, nums: List[int], pivot: int) -> List[int]:
        less=[]
        grater=[]
        ith=[]
        for i in range (len(nums)):
            if(nums[i]<pivot):
                less.append(nums[i])
            elif(nums[i]>pivot):
                grater.append(nums[i])
            else:
                ith.append(nums[i])
        return less+ith+grater

File: 2265-partition-array-according-to-given-pivot/2265-partition-array-according-to-given-pivot.cpp
class Solution {
public:
    vector<int> pivotArray(vector<int>& nums, int pivot) {
        
        vector<int>less;
        vector<int>grater;
        vector<int>ith;
        for(int i=0 ;i<nums.size();i++){
            if(nums[i]<pivot){
                less.push_back(nums[i]);
            }
            else if(nums[i]>pivot){
                grater.push_back(nums[i]);
            }
            else{
                ith.push_back(nums[i]);
            }
        }
        for(int i=0;i<ith.size();i++){
            less.push_back(ith[i]);
        }
        for(int i=0;i<grater.size();i++){
            less.push_back(grater[i]);
        }
        return less;
    }
};
        
